apps/rush/src/utilities/test/PackageChangeAnalyzer.test.ts

```diff
@@ -31,7 +31,7 @@ describe('PackageChangeAnalyzer', () => {
       }
     };
 
-    PackageChangeAnalyzer.getPackageDeps = (path: string, ignored: string[]) => repoHashDeps;
+    PackageChangeAnalyzer.getPackageDeps = (packagePath: string, ignored: string[]) => repoHashDeps;
```

Comment:
This violated the `no-shadowed-variable` TSLint rule.

---
In reply to: [129388208](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129388208) [](ancestors = 129388208)